CHANGELOG.md

```diff
@@ -9,15 +9,9 @@
 3.3.1 (unreleased)
```

Comment:
I am missing the list of new things which work or were changed (i.e. shadowserver config parser -> empty string -> by default select all reports.

Reply:
wait, I thought that's known bugs. Let me double check. Maybe I read the diffs too fast?

Damn, I did :) okay, then we are fine.